app/Http/Controllers/Api/V1/Integration/VideoDirectUrlController.php

```diff
@@ -109,6 +115,7 @@ private function getVimeoQualityVideo($videoArray, $quality)
     }
 
     /**
+     * Get Komodo direct url
```

Comment:
Add new line between title, description param and return.

Reply:
Done!